enterprise/utils.py

```diff
@@ -143,6 +143,34 @@ def get_idp_choices():
     return None
 
 
+def get_provider_login_url(request, provider_id):
```

Comment:
I think we should move this method in platform as it will reduce complexity and unnecessary imports.

Reply:
Already done.